lifecycle.md

```diff
@@ -46,13 +46,14 @@ For a mature standard, the whole process can repeat for proposed changes. New id
 
 As the standard, test suite and implementations co-evolve, an increasing number of tests will pass everywhere. For those tests, however trivial, full interop has been achieved. Once achieved, tooling should make it impossible to accidentally depart from full interop. Web developers can depend on the feature and delight their users, in any browser.
 
-## What about?
+## Examples
+ * DOM: mature [standard](https://dom.spec.whatwg.org/) and [test suite](https://github.com/w3c/web-platform-tests/tree/master/dom).
+ * HTML: the messagerror event had [spec text](https://github.com/whatwg/html/pull/2530) and [tests](https://github.com/w3c/web-platform-tests/pull/5567) written before any implementation.
```

Comment:
And critically before the change to the standard landed.

Reply:
OK, so here tests came before the spec text even. I'll update.

Reply:
Oh, actually it looks like it happened at the same time, per the now normal WHATWG working mode, and were merged together.

Reply:
Right, but I think that's rather critical as they end up influencing each other.

Reply:
In particular I want someone reading this to realize you end up writing a better specification.

Reply:
I guess that interplay happened for Fetch as well? Maybe I can say "Here and for the above case, the process of writing tests influenced the spec text and made it better."? Or is it important to also spell out the influence of spec text on tests, is that different when they happen together?